docs/release-notes/release-notes-0.8.0.md

```diff
@@ -37,10 +37,11 @@
 
 ## Functional Updates
 
-- [Garbage collection of zero-value UTXOs](https://github.com/lightninglabs/taproot-assets/pull/1832)
+- [Garbage collection of orphaned UTXOs](https://github.com/lightninglabs/taproot-assets/pull/1832)
```

Comment:
nit: merge this docs update with the previous docs commit, then move docs commit to the end so that it's the last one